packages/fluentui/docs/src/examples/components/Pill/Types/index.tsx

```diff
@@ -6,6 +6,11 @@ import ExampleSection from '../../../../components/ComponentDoc/ExampleSection';
 const Types = () => (
   <ExampleSection title="Types">
     <ComponentExample title="Default" description="A default Pill." examplePath="components/Pill/Types/PillExample" />
+    <ComponentExample
+      title="Pills"
+      description="A set of pills sematically wrapped by a Pills Container."
```

Comment:
nit: 
```suggestion
      description="A set of pills semantically wrapped by a Pills Container."
```

Also I don't really know what it's supposed to mean ?